src/app/hooks.ts

```diff
@@ -1,6 +1,34 @@
+import { useEffect, useState } from 'react';
 import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux';
 import type { RootState, AppDispatch } from './store';
 
+type WindowTypeSize = {
+    width: number | undefined;
```

Comment:
we can have just number here and below - as we are setting initial state it will never be undefined (if it is it means we fucked up sth really bad :D)

Reply:
yea I just forgot about it.